code/__DEFINES/preferences_defines.dm

```diff
@@ -81,6 +81,20 @@
 #error toggles_2 bitflag over 16777215. Please make an issue report and postpone the feature you are working on.
 #endif
 
+#define PREFTOGGLE_CATEGORY_GENERAL		1
```

Comment:
```suggestion
// This is a list index. Required to start at 1 instead of 0 so it's properly placed in the list
#define PREFTOGGLE_CATEGORY_GENERAL		1
```

Reply:
Works for me.